File: experiment/switch_ifelse_dict/ifelse.py
def applyTuple(action, num):
  if(action == 'half'):
    result = num / 2
  elif(action == 'double'):
    result = num * 2
  elif(action == 'triple'):
    result = num * 3
  elif(action == 'quadruple'):
    result = num * 4
  else:
    result = num

  return result

result = applyTuple('double', 2)

File: experiment/v1/switch_ifelse_dict/ifelse.js
function applyTuple(action, num) {
  var result;

  if(action === 'half') {
    result = num / 2;
  } else if(action === 'double') {
    result = num * 2;
  } else if(action === 'triple') {
    result = num * 3;
  } else if(action === 'quadruple') {
    result = num * 4;
  } else { 
    result = num;
  }
  
  return result;
}

var result = applyTuple('double', 2);